src/SmartComponents/Reports/Reports.js

```diff
@@ -57,9 +28,36 @@ const ReportsHeader = () => (
   </PageHeader>
 );
 
-export const Reports = () => {
+export const ReportsBase = ({ data, loading, error }) => {
+  const showView = data && data.length > 0;
```

Comment:
```suggestion
  const showView = data && data.length > 0;
```

is it correct to call `.length` on object here?

Reply:
the type of `data` is an array. Fixed in the propTypes.